applications/nrf_desktop/CMakeLists.txt

```diff
@@ -8,6 +8,9 @@ cmake_minimum_required(VERSION 3.20.0)
 
 ################################################################################
 
+# The application uses the configuration/<board> scheme for configuration files.
+set(APPLICATION_CONFIG_DIR "${CMAKE_CURRENT_LIST_DIR}/configuration/\${NORMALIZED_BOARD_TARGET}")
+
 find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
 project("nRF52 Desktop"
```

Comment:
Should the name of this project not change since now most devices for this sample are not nrf52's?

Reply:
Created a follow-up ticket for it - `NCSDK-29954`.
Not sure if we can modify this in all of the places - e.g. modifying device name in scan response data (and in scanning filters) would break backwards compatibility (new firmware would no longer be interoperable with the old firmware).